submission/persistence/src/main/kotlin/ac/uk/ebi/biostd/persistence/mapping/extended/to/ToExtSubmissionMapper.kt

```diff
@@ -19,7 +21,12 @@ private const val FILES_DIR = "Files"
 private const val USER_PREFIX = "u"
 
 class ToExtSubmissionMapper(private val submissionsPath: Path) {
```

Comment:
Unrelated typo in USER_PREFIX constant: it should be backward not background compatibility.